installation/scripts/install-ory.sh

```diff
@@ -0,0 +1,158 @@
+
+set -o errexit
+
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
+
+VALUES_FILE_ORY="${ROOT_PATH}"/chart/ory/values.yaml
+OVERRIDE_TEMP_ORY=ory-temp-values.yaml
+
+TIMEOUT=30m
+
+# Always create temporary override file, which is used whether "--overrides-file" is used or not
+cp "$VALUES_FILE_ORY" "$OVERRIDE_TEMP_ORY"
+
+# checkInputParameterValue is a function to check if input parameter is valid
+# There HAS to be provided argument:
+# $1 - value for input parameter
+# for example in installation/cmd/run.sh we can set --vm-driver argument, which has to have a value.
+
+function checkInputParameterValue() {
+    if [ -z "${1}" ] || [ "${1:0:2}" == "--" ]; then
+        echo "Wrong parameter value"
+        echo "Make sure parameter value is neither empty nor start with two hyphens"
+        exit 1
+    fi
+}
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case ${key} in
+        --overrides-file)
+            checkInputParameterValue "${2}"
+            yq eval-all --inplace '. as $item ireduce ({}; . * $item )' ${OVERRIDE_TEMP_ORY} ${2}
+            shift # past argument
+            shift
+        ;;
+        --timeout)
+            checkInputParameterValue "${2}"
+            TIMEOUT="${2}"
+            shift # past argument
+            shift
+        ;;
+        --skip-jwks-rotation)
+            # Skip the manual trigger of the Oathkeeper JWKS rotation CronJob
+            SKIP_JWKS_ROTATION=true
+            shift # past argument
+        ;;
+        --*)
+            echo "Unknown flag ${1}"
+            exit 1
+        ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+        ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+# Remove the temporary Ory values.yaml file created
+function cleanup_trap(){
+  if [[ -f "$OVERRIDE_TEMP_ORY" ]]; then
+    rm -f "$OVERRIDE_TEMP_ORY"
+  fi
+}
+
+# Generate random string with the length given as argument
+function generate_random(){
+  cat /dev/urandom | LC_ALL=C tr -dc 'a-z0-9' | fold -w ${1} | head -n 1
+}
+
+# Copy the IDP configuration from the Compass chart; the Compass values are changed by the `run.sh`
+VALUES_FILE_COMPASS="${ROOT_PATH}"/chart/compass/values.yaml
+IDP_HOST=$(yq ".global.cockpit.auth.idpHost" $VALUES_FILE_COMPASS)
+AUTH_PATH=$(yq ".global.cockpit.auth.path" $VALUES_FILE_COMPASS)
+
+# If IDP has been configured override the default values in the ORY chart values
+if [ ! -z "$IDP_HOST" ]; then
+  JWKS_URL=$IDP_HOST$AUTH_PATH
+
+  # Overwrite the jwks_url of the Ory Oathkeeper to match the Compass values
+  yq -i ".oathkeeper.oathkeeper.config.authenticators.jwt.config.jwks_urls = [\"$JWKS_URL\"]" "${OVERRIDE_TEMP_ORY}"
+fi
+
+trap cleanup_trap EXIT INT TERM
+
+echo "Helm install ORY components..."
+RELEASE_NS=ory
+RELEASE_NAME=ory-stack
+SECRET_NAME=ory-hydra-credentials
+
+kubectl create ns $RELEASE_NS || true
```

Comment:
A better way to create a namespace and not fail if it exists already is - `kubectl create ns $RELEASE_NS --dry-run=client -o yaml | kubectl apply -f -`. 
That way we won't have `Error from server (AlreadyExists): namespaces "ory" already exists` in the logs, instead we will have `namespace/ory configured`. Also, there is a tiny possibility where `kubectl create` fails for a different reason(not that the namespace already exists) and we will proceed further with the script which is not correct. This comment applies to the other scripts too(compass-gke-benchmark.sh)